Makefile

```diff
@@ -158,6 +158,11 @@ olm-deploy-bundle: bundle bundle-build
 .PHONY: olm-deploy-operator
 olm-deploy-operator: oci-build oci-push
 
+# Build and push the secret for the S3 storage
+.PHONY: olm-deploy-example-storage-secret
+olm-deploy-example-storage-secret:
+	sh hack/deploy-example-secret.sh $(CLUSTER_LOGGING_NS)
```

Comment:
nit. No need to prepend `sh` here, as Makefile's `SHELL` will work for that fine.